### ISSUE TITLE
All zero outputs only when running on Jetson Xavier NX

### ISSUE THREAD

**Issue by zhifeis** [triple-Mu/YOLOv8-TensorRT#88]
I first tried to convert and run my yolov8 model on Colab according to the steps in this repository and it worked perfectly. However, when I replicated the same steps on my Jetson Xavier NX device everything would run without error but the output is entirely zeros (i.e. no detections) when the same image produced high-confidence and accurate detections on Colab and on the base .pt model. Anyone know what's up with this?

**Comment by triple-Mu** [comment]
> I first tried to convert and run my yolov8 model on Colab according to the steps in this repository and it worked perfectly. However, when I replicated the same steps on my Jetson Xavier NX device everything would run without error but the output is entirely zeros (i.e. no detections) when the same image produced high-confidence and accurate detections on Colab and on the base .pt model. Anyone know what's up with this?

What's you trt version?
And suggest you convert pt to onnx on Colab and copy onnx to your Jetson, Then build engine by trtexec tools.

**Comment by zhifeis** [comment]
Yeah converting to onnx on Colab seemed to work for me, any idea why this is the case?

**Comment by triple-Mu** [comment]
Suggesting you using jetpack 4.6.3.
I just test on it on my TX2
